app/frontend/src/features/profile/profilePage/ProfilePage.tsx

```diff
@@ -20,15 +41,54 @@ const useStyles = makeStyles({
       textDecoration: "underline",
     },
   },
-});
+}));
+
+type DetailType = keyof typeof SECTION_LABELS;
 
 export default function ProfilePage() {
   const classes = useStyles();
   const { data: user } = useCurrentUser();
```

Comment:
If this page is meant to be shared for both the current user and viewing other users, this won't work 🤔  we'll probably need to delegate this to a consuming parent component and get them to pass in a user here instead.